tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "compilerOptions": {
       "declaration": true,
+      "esModuleInterop": true,
```

Comment:
hm... i would prefer we not do this and make the `import * as` statements instead. This will probably break upstream consumers who are not yet using `esModuleInterop`